enclave-runtime/Makefile

```diff
@@ -35,11 +35,11 @@ Rust_Enclave_Files := $(wildcard src/*.rs) $(wildcard ../stf/src/*.rs)
 RUSTFLAGS :="-C target-feature=+avx2"
 
 ifeq ($(SGX_DEBUG), 1)
-	OUTPUT_PATH := release
-	CARGO_TARGET := --release
+	OUTPUT_PATH := debug
+	CARGO_TARGET := 
 else
-	OUTPUT_PATH := release
-	CARGO_TARGET := --release
+	OUTPUT_PATH := debug
```

Comment:
Wait, we stay in release here, don't we?

Reply:
Absolutely, nice catch!
(Global search and replace may not be the best idea sometimes :D)

Reply:
Hahah, I actually very seldom use it just because of these caveats.